core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaStore.java

```diff
@@ -279,9 +283,27 @@ private void verifySchemaTopic(AdminClient admin) throws StoreInitializationExce
    */
   public void waitUntilKafkaReaderReachesLastOffset(int timeoutMs) throws StoreException {
     long offsetOfLastMessage = getLatestOffset(timeoutMs);
-    log.info("Wait to catch up until the offset of the last message at " + offsetOfLastMessage);
-    kafkaTopicReader.waitUntilOffset(offsetOfLastMessage, timeoutMs, TimeUnit.MILLISECONDS);
-    log.debug("Reached offset at " + offsetOfLastMessage);
+    waitUntilKafkaReaderReachesOffset(offsetOfLastMessage, timeoutMs);
+  }
+
+  /**
+   * Wait until the KafkaStore catches up to the given subject offset in the Kafka topic.
+   */
+  public void waitUntilKafkaReaderReachesLastSubjectOffset(String subject, int timeoutMs)
```

Comment:
subject looks redundant in the name
```suggestion
  public void waitUntilKafkaReaderReachesLastOffset(String subject, int timeoutMs)
```